### DISCUSSION TITLE
bot online 24/7

### DISCUSSION THREAD

**Discussion by Bruhman69420XD** [machineonamission/mediaforge#113] [Q&A] [answered]
why can't a bot be online 24/7

**Comment by machineonamission** [comment, marked as answer]
For discord to consider a bot online, it must be continually communicating with discord. Unfortunately, the current code of MediaForge is poorly optimized and struggles to run at the scale it’s at, constantly overloading the PC it’s running on and interrupting its connection to Discord. This will Eventually be fixed when I rewrite the processing code but I’m busy and that’s a big task so

**Reply by Bruhman69420XD** [reply]
ok thanks

**Reply by Bruhman69420XD** [reply]
but what time will it bee online again

**Reply by machineonamission** [reply]
I’m trying to restart it regularly when it goes down
